### ISSUE TITLE
How to disable IPV6 access?

### ISSUE THREAD

**Issue by riku22** [jmdaweb/NVDARemoteServer#26]
Hello.
I have set up an NVDA remote server on Ubuntu.
IPV6 access is disabled on my server.
I don't know if it is because of this, but when starting NVDA remote server, the following error is output to the log and it fails to start.
Is there a way to disable the use of IPV6 on NVDA remote server?

```
Loggin system initialized.
Sun Mar 22 04:00:14 2020
Configuring signal handlers
Sun Mar 22 04:00:14 2020
Initialized instance variables
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/share/NVDARemoteServer/server.py", line 234, in run
    self.createServerSocket(self.port, self.port6, self.bind_host, self.bind_host6)
  File "/usr/share/NVDARemoteServer/server.py", line 205, in createServerSocket
    self.server_socket6 = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
  File "/usr/lib/python2.7/socket.py", line 191, in __init__
    _sock = _realsocket(family, type, proto)
error: [Errno 97] Address family not supported by protocol

Closing logger thread...

```

**Comment by jmdaweb** [comment]
Hi,
In the latest commit, the server tries to initialize support for both IPV6 and IPV4, and it only fails if no sockets can be created for any of them. The new version will be released soon. Sorry for the delay.
